library/src/skins/default/src/sass/base/_jquery-ui-defaults.scss

```diff
@@ -30,8 +30,11 @@
         font-size: 1em;
     }
 
+    // --sakai-border-color is close to --sakai-background-color-1 so the overall widget
+    // does not stand out - to make the overall widget border "pop" we make it the same
+    // color as the text in the Sakai theme
     .ui-widget.ui-widget-content {
-        border: 1px solid var(--sakai-border-color);
+        border: 2px solid var(--sakai-text-color-1);
```

Comment:
What if update the --sakai-border-color value for better contrast?

- (Dark Scss file) https://github.com/sakaiproject/sakai/blob/c4c788a69e7d6b56a1f2250bb720c050625b1b40/library/src/skins/default/src/sass/themes/_dark.scss#L135

- (Light Scss file)https://github.com/sakaiproject/sakai/blob/c4c788a69e7d6b56a1f2250bb720c050625b1b40/library/src/skins/default/src/sass/themes/_light.scss#L144